actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/validator/user/UserBulkMigrationRequestValidator.java

```diff
@@ -177,6 +214,23 @@ private void checkInputStatus(String inputStatus, int index) {
     }
   }
 
+  private void checkSelfDeclaredInputStatus(String inputStatus, int index) {
+    CsvRowErrorDetails errorDetails = new CsvRowErrorDetails();
+    errorDetails.setRowId(index);
+    if (StringUtils.isBlank(inputStatus)) {
+      errorDetails.setHeader(JsonKey.INPUT_STATUS);
+      errorDetails.setErrorEnum(ErrorEnum.missing);
+      addErrorToList(errorDetails);
+
+    } else if (!(inputStatus.equalsIgnoreCase(JsonKey.PENDING)
```

Comment:
Please create an enum for pending, rejected, etc. This way we can parse the string to an enum or not and fail easily if no matches if found.
Just imagine if there are 10 status flag - we keep adding 10 or conditions, eh?

Reply:
@indrajra : Moved those status to enum.